labonneboite/web/api/views.py

```diff
@@ -601,8 +602,11 @@ def get_office_details(siret, alternance=False):
 def patch_office_result_with_sensitive_information(api_username, office, result, alternance=False):
     # Some internal services of Pôle emploi can sometimes have access to
     # sensitive information.
-    if api_username in settings.API_INTERNAL_CONSUMERS:
-        result['email'] = office.email_alternance if alternance and office.email_alternance else office.email
-        result['phone'] = office.phone_alternance if alternance and office.phone_alternance else office.tel
-        result['website'] = office.website_alternance if alternance and office.website_alternance else office.website
+    scopes = user.get_api_user_scopes(api_username)
+    result['scopes'] = [scope.name for scope in scopes]
+    if(Scope.COMPANY_EMAIL in scopes): result['email'] = office.email_alternance if alternance and office.email_alternance else office.email
+    if(Scope.COMPANY_PHONE in scopes): result['phone'] = office.phone_alternance if alternance and office.phone_alternance else office.tel
+    if(Scope.COMPANY_WEBSITE in scopes): result['website'] = office.website_alternance if alternance and office.website_alternance else office.website
+    if(Scope.COMPANY_PMSMP in scopes): result['pmsmp'] = office.flag_pmsmp
+    if(Scope.COMPANY_POE in scopes): result['poe'] = office.flag_poe_afpr
```

Comment:
poe != boe
we need boe here (handicap)
and not poe (afpr/poe/psmsp == imersion)